modules/project_cleanup/variables.tf

```
@@ -77,6 +77,18 @@ variable "target_included_labels" {
   default     = {}
 }
 
+variable "clean_up_org_level_tag_keys" {
+  type        = bool
+  description = "Clean up organization level Tag Keys."
+  default     = false
+}
+
+variable "target_excluded_tagkeys" {
+  type        = list(string)
+  description = "List of organization Tag Keys short names that won't be deleted."
```

As "names" is plural, I'm not sure if we also need the "s" on "Key"?

```suggestion
  description = "List of organization Tag Key short names that won't be deleted."
```

---

fixed